library/edi_libre.inc

```diff
@@ -0,0 +1,538 @@
+<?php
```

Comment:
Should this file be name edi_libre.inc?   Seems like it should be named in a way that references it purpose.

edi_276.inc.php perhaps.